Civil-Service-Human-Resources/rpg-careers-wp
# 396 add theme files to this repo

__What__

This PR adds the Theme files from the old wp repo into a new folder called theme.

__How to test__

1. Check that the theme files have been added to the theme folder correctly.
2. Check that the correct git commits have been maintained across the the two project.s
